tests/test_util.go

```diff
@@ -297,3 +297,69 @@ func removeSecret(cli kubernetes.Interface, secretName, ns string) error {
 	}
 	return nil
 }
+
+func arangoDeploymentHealthy(t *testing.T, deployment *api.ArangoDeployment, k8sClient kubernetes.Interface) {
```

Comment:
Name is now clean to me.
Suggest to rename to `waitUntilArangoDeploymentHealthy`